### PR TITLE
ClientDriven Sample - Welcome Dialog and Table of Contents [MTT-8693] [MTT-8697]

### PR DESCRIPTION
**Pull request by FabianStoll**
### Description
Adds the standard Welcome Dialog and Table of Contents from the in-editor tutorials package to the ClientDriven Sample.

### Issue Number(s)
https://jira.unity3d.com/browse/MTT-8693
https://jira.unity3d.com/browse/MTT-8697

### Contribution checklist
 - [ ] Tests have been added for the project and/or any internal package
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink

### CODE REVIEW

File: Basic/ClientDriven/Assets/Editor/Tutorials/TableOfContents.asset [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    m_Untranslated: 
+  ProjectLayout: {fileID: 0}
+  Sections:
+  - OrderInView: 1
+    Heading:
+      m_Untranslated: Dedicated Forum
+    Text:
+      m_Untranslated: Ask questions and get help on Netcode for GameObjects
+    Metadata: 
+    Url: https://forum.unity.com/forums/netcode-for-gameobjects.661/
```
Comment (bastienunity) [inline Basic/ClientDriven/Assets/Editor/Tutorials/TableOfContents.asset:35]: This is the link to the old forum, it has a redirect but maybe we should link to discussion directly
https://discussions.unity.com/tags/c/unity-engine/52/netcode-for-gameobjects

**Review by bastienunity — APPROVED** [review]
I added a suggestion for the forum link, but that's probably fine to land without it.
Regarding the icons, @selenaayang did a pass on the new Template with updated designs, maybe you'd like to re-use them in the sample too: https://jira.unity3d.com/browse/MTT-9640, but again, not blocking if we want to land this PR.

**Review by bastienunity — COMMENTED** [review]
I think the icons path should be clean to avoid any unforeseen issues with strange characters in the Assets folder.

File: Basic/ClientDriven/ProjectSettings/Packages/com.unity.learn.iet-framework/Settings.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
             {
                 "type": "System.Boolean, mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089",
                 "key": "IET.DisplayWelcomeDialogOnStartup",
-                "value": "{\"m_Value\":false}"
+                "value": "{\"m_Value\":true}"
```
Comment (RikuTheFuffs) [inline Basic/ClientDriven/ProjectSettings/Packages/com.unity.learn.iet-framework/Settings.json:7]: nice catch!

**Review by bastienunity — APPROVED** [review]
Looks all good to me!

**Pull request merged**